src/cmd/flux-account-service.py

```diff
@@ -133,6 +133,8 @@ def view_user(self, handle, watcher, msg, arg):
             handle.respond(msg, payload)
         except KeyError as exc:
             handle.respond_error(msg, 0, f"missing key in payload: {exc}")
+        except ValueError as val_err:
+            handle.respond_error(msg, 0, f"error in view-user: {str(val_err)}")
```

Comment:
I don't think `str(val_err}` is required here, just `{val_err}` should be sufficient

Reply:
You are right, good catch, thank you! `{val_err}` is sufficient here. I just force-pushed up a fix to just use `{val_err}`